doc/development_process/index.rst

```diff
@@ -0,0 +1,310 @@
+
+The LTS is released every 2 years.
+
+An LTS release will be branched and maintained independently of the mainline
+tree.
+
+
+.. figure:: lts.png
+    :align: center
+    :alt: Long Term Support Release
+    :figclass: align-center
+
+    Long Term Support Release
+
+Changes and fixes flow in both directions, however, changes from master to
+LTS branch will be limited to fixes that apply to both branches and for existing
+features only.
+
+All fixes for LTS that apply to the mainline tree are pushed to
+mainline as well.
+
+Development Environment and Tools
+*********************************
+
+Code Review
+============
+
+GitHub is intended to provide a framework for reviewing every commit before it
+is accepted into the code base. Changes, in the form of Pull Requests (PR) are
+uploaded to GitHub but don’t actually become a part of the project until they’ve
+been reviewed, passed a series of checks (CI) and approved by maintainers.
+GitHub is used to support the standard open source practice of submitting
+patches, which are then reviewed by the project members before being applied to
+the code base.
+
+The Zephyr project uses GitHub for code reviews and GIT tree management. When
+submitting a change or an enhancement to any Zephyr component, a developer should
+use GitHub. GitHub automatically assigns a responsible reviewer on a component
+basis, as defined in the CODEOWNERS file stored with the code tree in the Zephyr
+project repository. A limited set of release managers are allowed to merge a PR
+into the master branch once reviews are complete.
+
+Continuous Integration
+=======================
+
+All changes submitted to GitHub are subject to sanity tests that are run on
+emulated platforms and architectures to identify breakage and regressions that
+can be immediately identified.  Sanity testing additionally performs build tests
+of a representative number of boards and platforms (subject to hardware
+availability in the CI infrastructure). Documentation changes are also verified
+through review and build testing to verify doc generation will be successful.
+
+Any failures found during the CI test run will result in a negative review.
+Developers are expected to fix issues and rework their patches and submit again.
+
+The CI infrastructure currently runs the following tests:
+
+- Run '''checkpatch''' for code style issues (can vote -1 on errors)
+- Gitlint: Git commit style based on project requorements
+- License Check: Check for conflicting licenses
+- Run '''sanitycheck''' script:
+
+  - Run kernel tests in QEMU (can vote -1 on errors)
+  - Build various samples for different boards (can vote -1 on errors)
+
+
+Dealing with Proposals and RFCs
+*******************************
+
+Feature or enhancement proposals for Zephyr are used to propose a new design or
+modify an existing feature and get it approved. Currently, the developer mailing
+list is used to make such proposals. To better integrate with the current tools
```

Comment:
"Currently, the developer mailing list is used to make such proposals"
I guess this is not anymore the case?